avocado/plugins/replay.py

```diff
@@ -9,19 +9,22 @@
 #
 # See LICENSE for more details.
 #
-# Copyright: Red Hat Inc. 2013-2014
-# Author: Lucas Meneghel Rodrigues <lmr@redhat.com>
+# Copyright: Red Hat Inc. 2016
+# Author: Amador Pahim <apahim@redhat.com>
```

Comment:
This should be a different commit with a proper explanation (which is I guess copy&paste issue as the file did not exists in 2014)